CommandHelper/LocalPackages/commands/powers/skeletonarmy.msa

```diff
@@ -0,0 +1,22 @@
+skeletonarmy:/skeletonarmy = >>>
+
+    @player = player()
+
+    if(!is_null(import(@player."skeletonarmy"))){
+        @oldskellys = import(@player."skeletonarmy")
+        foreach(@uuid in @skellys){
+            try(entity_remove(@uuid),
+            continue())
```

Comment:
continue() not necessary as loop will continue naturally. However it doesn't break the code so technically not wrong, just not needed.